node/sc/bridge_manager_test.go

```diff
@@ -488,32 +469,13 @@ func TestBridgeManagerWithFee(t *testing.T) {
 					"requestNonce", ev.RequestNonce,
 					"fee", ev.Fee.String())
 
-				switch ev.TokenType {
-				case KLAY:
-					assert.Equal(t, common.Address{}, ev.TokenAddress)
-					assert.Equal(t, KLAYFee, ev.Fee.Int64())
-
-					// HandleKLAYTransfer by Event
-					tx, err := pBridge.HandleKLAYTransfer(&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit}, ev.Raw.TxHash, ev.From, ev.To, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleKLAYTransfer: %v", err)
-					}
-					fmt.Println("HandleKLAYTransfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-
-				case ERC20:
-					assert.Equal(t, tokenAddr, ev.TokenAddress)
-					assert.Equal(t, ERC20Fee, ev.Fee.Int64())
-
-					// HandleERC20Transfer by Event
-					tx, err := pBridge.HandleERC20Transfer(&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit}, ev.Raw.TxHash, ev.From, ev.To, tokenAddr, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleERC20Transfer: %v", err)
-					}
-					fmt.Println("HandleERC20Transfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-				}
+				// insert the value transfer request event to the bridge info's event list.
+				pBridgeInfo.AddRequestValueTransferEvents([]*RequestValueTransferEvent{ev})
+
+				// handle the value transfer request event in the event list.
+				pBridgeInfo.processingPendingRequestEvents()
 
+				sim.Commit() // block
```

Comment:
Question: what is meaning of the `// block` ?

Reply:
It means mining block. :)